tasks/main.yml

```diff
@@ -4,13 +4,13 @@
 
 - import_tasks: el7/el7_tasks.yml
   when:
-    - ansible_distribution == 'CentOS'
-    - ansible_distribution_major_version == '7'
+    - (ansible_facts['distribution'] == "CentOS" and ansible_facts['distribution_major_version'] == "7") or
+      (ansible_facts['distribution'] == "RedHat" and ansible_facts['distribution_major_version'] == "7")
 
 - import_tasks: el8/el8_tasks.yml
   when:
-    - ansible_distribution == 'CentOS'
```

Comment:
You could simplify this a bit with contains check.
```

when:
  - ansible_facts['distribution'] in ["CentOS", "RedHat"]
  - ansible_facts['distribution_major_version'] == "8"
```

Reply:
My intent was to decouple the two checks for the individual OSes, but this definitely reads better. Fixed.